azure-pipelines.yml

```diff
@@ -0,0 +1,422 @@
+pool:
```

Comment:
Is it possible for this to be `.github/azure-pipelines.yml`?

Reply:
Apparently it's _possible_, but very non-standard, and apparently requires a counterintuitive process to set up. https://stackoverflow.com/questions/55614307/can-you-change-the-location-of-azure-pipelines-yaml-in-azure-devops

It looks like CPython have their config in `.azure-pipelines/ci.yml`, so may be worth looking into how exactly that's set up there.